packages/checkout/widgets-lib/src/views/top-up/TopUpView.tsx

```diff
@@ -57,6 +58,11 @@ interface TopUpViewProps {
   subheading?: [key: string, options?: $Dictionary];
 }
 
+export const TOOLKIT_BASE_URL = {
+  [Environment.SANDBOX]: 'https://checkout-playground.sandbox.immutable.com',
+  [Environment.PRODUCTION]: 'https://toolkit.immutable.com',
+};
+
```

Comment:
we have a config file under `ts-immutable-sdk/packages/checkout/widgets-lib/src/widgets/sale/utils/config.ts` that we can move this to 👀

Reply:
Is there a similar place (like `packages/checkout/widgets-lib/src/lib/utils.ts`) outside of the sale scope that would be suitable? I was conflicted with this one since it's not necessarily sale specific

Reply:
ah true, `TopUpView` is outside of the `/widgets` folder - happy for it to stay here for now!